src/tools/m65.c

```diff
@@ -48,6 +48,10 @@
 
 #include "m65common.h"
 
+#define UT_TIMEOUT 10
```

Comment:
This is fine for now, but we should probably have a mechanism for changing this. This could be from the test side using one of the unallocated codes to allow the test program to communicate the new timeout.